app/assets/stylesheets/modules/item-selector.scss

```diff
@@ -86,3 +86,34 @@
   }
 }
 
+.bound-withs {
+  max-height: 200px;
+  background-color: #F4F4F4;
```

Comment:
Please make this a variable and put it in https://github.com/sul-dlss/sul-requests/blob/main/app/assets/stylesheets/sul-variables.scss#L1